tests/curried-utils/test_curried.py

```diff
@@ -46,6 +46,8 @@ def test_curried_namespace():
     def should_curry(func):
         if not callable(func) or isinstance(func, curry):
             return False
+        if getattr(func, '__module__', None) == eth_utils.exceptions.__name__:
+            return False
```

Comment:
Not sure if there's a better way to achieve this? Also, is it correct to assume that exceptions shouldn't be curry-able?

Reply:
This is fine by me. cc @carver for final approval since this was his original creation.  (i'd also be up for removing the *magic* and making this explicit and thus, being able to remove this entire test as well.

Reply:
Yeah, this is fine. :+1:  An alternative:

```py
if isinstance(func, Exception):
  return False
```

I agree that exceptions shouldn't be curriable.

I think there's some value in the test, to make sure someone thinks about whether a method *should* be curriable when it's added to eth-utils.

Reply:
@carver I tried that, but for some reason it doesn't work as expected and I can't quite put my finger on why not - checking the module names was the only solution I could find that would work.

Reply:
Because it's not an instance. :P

```py
if issubclass(func, Exception):
  return False
```

Reply:
beware, you'll need to do `if isinstance(func, type) and issubclass(func, Exception)`.  Annoyingly, `issubclass` disallows passing in non-class values.